jcl/src/openj9.dtfj/share/classes/com/ibm/dtfj/image/j9/ImageFactory.java

```diff
@@ -93,56 +108,57 @@
 	 * This class will generally be referred to by name (e.g. using Class.forName()).
 	 * 
 	 * @see com.ibm.dtfj.image.ImageFactory
-	 *
 	 */
 	public ImageFactory() {
-		//when the vm shuts down we need to make sure that all our extracted files are deleted
+		// when the vm shuts down we need to make sure that all our extracted files are deleted
 	}
 
+	@Override
 	public Image[] getImagesFromArchive(File archive, boolean extract) throws IOException {
-		if (archive != null && !FileManager.fileExists(archive)) {
-			throw new FileNotFoundException("Archive '" + archive.getAbsolutePath() + "' not found.");
+		Objects.requireNonNull(archive);
```

Comment:
Should change the javadoc to mention NPE.

Reply:
While the javadoc (for the interface) does not explicitly say null is *not* permitted, it is my understanding that one should assume null is not acceptable unless the documentation explicitly says it *is* permitted.

The old version threw NPEs under the same circumstances, it's just clearer now.

Reply:
I still think the javadoc should be updated, but I won't hold up this PR for it.